prow/plugins.yaml

```diff
@@ -41,6 +41,8 @@ plugins:
   - shrug
   - wip
   - yuks
+  - hold
+  - needs-rebase
```

Comment:
`needs-rebase` is an external plugin, not a built in plugin. I suggest omitting the `needs-rebase` plugin for now.